debian/rules

```diff
@@ -19,6 +19,12 @@ DEB_CMAKE_EXTRA_FLAGS := \
 	-DWITH_SYSVINIT=ON \
 	-DWITH_SYSTEMD=$(WITH_SYSTEMD)
 
+# Append CMAKE flag with -DLUAJIT_ENABLE_GC64 value. It needs for CI
+# to build Tarantool with gc64 enabled or disabled dynamically.
+ifneq "$(GC64)" ""
```

Comment:
Looks like you can use `ifdef GC64` here. It is up to you.

Reply:
replaced with a common syntax `ifneq ($(GC64),)`